spec/support/features/doc_auth_helper.rb

```diff
@@ -227,6 +227,10 @@ def enable_doc_auth
     allow(FeatureManagement).to receive(:doc_auth_exclusive?).and_return(true)
   end
 
+  def enable_acuant_simulator(enabled)
```

Comment:
`enable_acuant_simulator` seems like kind of a confusing name given that in 50% of cases this method doesn't enable the acuant simulator. What about something like `setup_acuant_simulator(enabed: true)`? That may make it a bit more clear what it's doing in the test.